test_utilities/src/aws.rs

```diff
@@ -800,3 +800,53 @@ pub fn environment_only_http_server(context: &Context) -> Environment {
     }
 }
 
+pub fn environment_only_http_server_router(context: &Context) -> Environment {
```

Comment:
What is this? If this is the "minimum", shouldn't we just update the current existing one instead?